tests/app-pip/00_urllib.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Why not invoke the Python file directly? We can add a Python3 shebang line: `#!/usr/bin/python3`

Reply:
Not sure about this one, since with the test we are also indirectly testing the `/app/main.py` structure

Reply:
Using python allows us to check if the python version which is in the path is working

Reply:
Discussed in private, going to leave it like this for now.